tests/integration/test_backward_compatibility/test_block_marshalling.py

```diff
@@ -0,0 +1,113 @@
+import pytest
+
+from helpers.cluster import ClickHouseCluster
+
+cluster = ClickHouseCluster(__file__)
+cluster_name = "parallel_replicas"
+nodes = [
+    cluster.add_instance(
+        f"node{num}",
+        main_configs=["configs/clusters.xml"],
+        with_zookeeper=False,
+        image="library/clickhouse",
+        tag="25.1",
+        stay_alive=True,
+        use_old_analyzer=True,
```

Comment:
Maybe better do it for a particular query by setting `enable_analyzer` to be more explicit in intention?